lib/libc/common/source/time/clock.c

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <time.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys_clock.h>
+
+#ifndef CLOCKS_PER_SEC
+#warning libc does not define CLOCKS_PER_SEC which is required by ISO
```

Comment:
I'd say this is an error -- applications cannot use 'clock' without this value; we can catch that at compile time right here.